shell-ui/src/setupTests.js

```diff
@@ -2,6 +2,36 @@ import 'whatwg-fetch';
 import "regenerator-runtime/runtime"
 import '@testing-library/jest-dom/extend-expect';
 
+if (!Array.prototype.flat) {
+	Object.defineProperty(Array.prototype, 'flat', {
+		configurable: true,
+		value: function flat () {
+			var depth = isNaN(arguments[0]) ? 1 : Number(arguments[0]);
+
+			return depth ? Array.prototype.reduce.call(this, function (acc, cur) {
+				if (Array.isArray(cur)) {
+					acc.push.apply(acc, flat.call(cur, depth - 1));
+				} else {
+					acc.push(cur);
+				}
+
+				return acc;
+			}, []) : Array.prototype.slice.call(this);
+		},
+		writable: true
+	});
+}
+
+if (!Array.prototype.flatMap) {
+	Object.defineProperty(Array.prototype, 'flatMap', {
+		configurable: true,
+		value: function flatMap (callback) {
+			return Array.prototype.map.apply(this, arguments).flat();
+		},
+		writable: true
+	});
+}
```

Comment:
Why do we need those polyfills? I don't even see those methods used in the code :/

Reply:
I was using flatMap at some point in my implementation and as tests runs using node 10 it was not available, I will clean this up 👍

Reply:
Argh, but since this project (shell-ui) is separate from the main MetalK8s UI, why aren't we already using Node 14.x? :wink: